<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageAsmGenerationWidget</class>
 <widget class="QWidget" name="LanguageAsmGenerationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkNoDebugDirectives">
     <property name="text">
      <string>Don't generate debug directives</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNoSEHDirectives">
     <property name="text">
      <string>Don't generate SEH directives </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpX86Syntax">
     <property name="title">
      <string>Instruction syntax:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbATT">
        <property name="text">
         <string>AT&amp;&amp;T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbIntel">
        <property name="text">
         <string>Intel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
